docs/internals/sync.md

```diff
@@ -20,25 +20,26 @@ In the context of state sync, "state" refers to:
 6. the grid (LSM data; acquired blocks only)
 7. client replies
 
-However, state sync protocol itself only syncs the first 5.
-6/7 are replicated after state sync protocol, driven by grid repair and the disk scrubber.
+State sync consists of two protocols:
+- [Sync Superblock](./vsr.md#protocol-sync-superblock) (syncs 1-5)
+- [Sync Content](./vsr.md#protocol-sync-content) (syncs 6-7)
```

Comment:
```suggestion
- [Sync Grid](./vsr.md#protocol-sync-grid) (syncs 6-7)
```

"Content" was not clear/descriptive enough to me at first glance, to get an idea as to what might be synced. "Content" is more general, like "Data". Whereas, if we are syncing the grid blocks here, then I think it would be great to call it that, rather than introduce a new term.

Reply:
Yes, I noticed that. Sync Client Replies could also be decoupled out of that.

Reply:
Then the different sync strategies all relate to the zone they sync.